apps/microsoft-teams/frontend/src/hooks/useGetContentTypes.spec.ts

```diff
@@ -26,5 +33,21 @@ describe('useGetContentTypes', () => {
       expect(result.current.error?.message).toEqual('Unable to get content types');
       expect(result.current.loading).toEqual(false);
     });
+
+    vi.restoreAllMocks();
+  });
+
+  it('batches requests if the number of available content types exceeds the request limit', async () => {
+    const getManyResponse = cloneDeep(mockGetManyContentType);
```

Comment:
Small thing but these days I think cloning via es6 destructuring is just as easy as lodash and fewer unnecessary external dependencies.

```js
const getManyResponse = { ...mockGetManyContentType }
```

Reply:
Yeah, I like destructuring for accessing arrays/objects, but with objects (and maybe array?) it's not a _true_ clone utility, in the sense that nested object properties for the cloned object, also get updated for the object that was cloned, and vice versa.

```js
const a = { abc: 1, nested: { def: "def" } }
const b = { ...a }
b.abc = 2
console.log(a)
// { abc: 1, nested: { def: "def" } }. 👍 no issues!

b.nested.def = "uh oh!!"
console.log(a)
// { abc: 1, nested: { def: "uh oh!!" } } a.nested.def was updated.
```

So when I actually want to deep (any nested props) clone, I like lodash cloneDeep or even `JSON.parse(JSON.stringify(foo))`.